types.go

```diff
@@ -1247,6 +1278,185 @@ func (i *Float128) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// added 2022-11-02
```

Comment:
No need for this for code you are adding that is unrelated to version of EOSIO/LEAP

Reply:
https://github.com/eoscanada/eos-go/pull/191/commits/18d78b47dd7be8c506ce92f5422a4273eb0d1820